src/front-door/azext_front_door/_params.py

```diff
@@ -129,15 +129,17 @@ def load_arguments(self, _):
     with self.argument_context('network front-door routing-rule', arg_group=None) as c:
         c.argument('accepted_protocols', nargs='+', help='Space-separated list of protocols to accept. Default: Http')
         c.argument('patterns_to_match', options_list='--patterns', nargs='+', help='Space-separated list of patterns to match. Default: \'/*\'.')
-        c.argument('rules_engine', help='Name or ID of a Rules Engine configuration.', validator=validate_rules_engine)
+        c.argument('rules_engine', help='Name or ID of a Rules Engine configuration. To unlink property, \"--remove rulesEngine\"', validator=validate_rules_engine)
         c.argument('frontend_endpoints', help='Space-separated list of frontend endpoint names or IDs.', nargs='+', validator=validate_frontend_endpoints)
     with self.argument_context('network front-door routing-rule', arg_group='Forward Routing Rule') as c:
         c.argument('backend_pool', help="Name or ID of a backend pool. It's required to create a Forward routing rule.", validator=validate_backend_pool)
-        c.argument('forwarding_protocol', help='Protocol to use for forwarding traffic.')
+        c.argument('forwarding_protocol', arg_type=get_enum_type(FrontDoorForwardingProtocol), help='Protocol to use for forwarding traffic.')
         c.argument('custom_forwarding_path', help='Custom path used to rewrite resource paths matched by this rule. Leave empty to use incoming path.')
         c.argument('caching', arg_type=get_three_state_flag(positive_label='Enabled', negative_label='Disabled', return_label=False), help='Whether to enable caching for this route.')
+        c.argument('cache_duration', help='The duration for which the content needs to be cached. Allowed format is ISO 8601 duration')
```

Comment:
what's the unit for duration? seconds? minutes? better to add it in help so users can understand it directly.

Reply:
I have mentioned that the duration is ISO 8601 (self explanatory) as it is widely used, it's like P1DT12H30M30S means 1 day, 12 hours, 30 minutes, 30 seconds.